packages/front-end/components/Features/CodeSnippetModal.tsx

```diff
@@ -34,11 +34,12 @@ export function getApiBaseUrl(connection?: SDKConnectionInterface): string {
     );
   }
 
+  //MKTODO: We should be able to remove this if we add an env variable to our cloud instance
   if (isCloud()) {
     return `https://cdn.growthbook.io`;
   }
 
-  return trimTrailingSlash(getApiHost());
+  return getCdnHost() || trimTrailingSlash(getApiHost());
```

Comment:
I think we would want to call `trimTrailingSlash` not matter which env variable is being read.

Reply:
Good call. Updated.